services/crypto_service.proto

```diff
@@ -62,6 +62,11 @@ service CryptoService {
      */
     rpc deleteAllowances (Transaction) returns (TransactionResponse);
 
+    /**
+     * Generates a pseudorandom number.
+     */
+    rpc randomGenerate (Transaction) returns (TransactionResponse);
```

Comment:
Why is this being associated with the crypto service? That decision seems a bit random (sorry for the pun).

Perhaps this should be a network_service or possible a new type of util service?

Reply:
Added `UtilService`